### PR TITLE
Update svg emoji tools.

### PR DESCRIPTION
**Pull request by dougfelt**
- Reformat lines to 80 columns.

- Use logging instead of verbose/quiet other options.

- A few miscellaneous small fixes/tweaks to parameters.  Removed some
  file-path-relative stuff that assumed old directory structure.

This uses some new fns in nototools.tool_utils, see nototools#220.

### CODE REVIEW

File: collect_emoji_svg.py [hunk 1]
```diff
@@ -75,66 +82,69 @@ def copy_with_rename(src_dir, dst_dir, accept_pred=None, rename=None, verbosity=
...
     shutil.copy2(src, dst)
-    if verbosity > 1:
-      print "cp -p %s %s" % (src, dst)
+    logging.debug('cp -p %s %s', src, dst)
     count += 1
-  if verbosity:
-    print "Copied/renamed %d files from %s to %s" % (count, src_dir, dst_dir)
-  return count, replace_count
-
+  if logging.getLogger().getEffectiveLevel() >= 20:
```
Comment (jamesgk) [inline collect_emoji_svg.py]: 20 feels like kind of a magic number here, could we instead do `>= logging.INFO`?

File: svg_cleaner.py [hunk 2]
```diff
@@ -230,25 +254,45 @@ def clean_svg_files(in_dir, out_dir, match_pat=None, quiet=False):
...
+  parser.add_argument(
+      '-o', '--out_dir', help='Output directory, defaults to sibling of in_dir',
+      metavar='dir')
+  parser.add_argument(
+      '-c', '--clean', help='Clean output directory', action='store_true')
+  parser.add_argument(
+      '-r', '--regex', help='Regex to select files, default matches all files.',
+      metavar='regex', default=None)
+  parser.add_argument(
+      '-q', '--quiet', dest='v', help='Quiet operation.', default=1,
```
Comment (jamesgk) [inline svg_cleaner.py]: No longer used?

**Comment by jamesgk** [comment]
LGTM

File: svg_cleaner.py (hunk 2 above)
Comment (dougfelt) [inline svg_cleaner.py]: Oops, forgot to delete this.

File: collect_emoji_svg.py (hunk 1 above)
Comment (dougfelt) [inline collect_emoji_svg.py]: Yes.  Also, it should have been <=

**Pull request merged**